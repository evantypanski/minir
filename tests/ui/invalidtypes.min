func main() -> none {
  let i: notatype = 0
  ret
}

func f() -> notatype {
  ret
}

func f2(i: notatype) -> none {
  ret
}
