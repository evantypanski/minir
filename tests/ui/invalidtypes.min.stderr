error: at tests/ui/invalidtypes.min:2: 'notatype' is not a valid type name
  let i: notatype = 0;
error: at tests/ui/invalidtypes.min:6: 'notatype' is not a valid type name
func f() -> notatype {
error: at tests/ui/invalidtypes.min:10: 'notatype' is not a valid type name
func f2(i: notatype) -> none {


Found 3 errors during parsing
error: Fatal error occurred; aborting
